libbeat/scripts/generate_fields_docs.py

```diff
@@ -130,6 +130,8 @@ def fields_to_asciidoc(input, output, beat):
 This file is generated! See _meta/fields.yml and scripts/generate_fields_docs.py
 ////
 
+:edit_url:
```

Comment:
Is the point of this just to turn off the edit links in the fields docs?

Reply:
That's right! We definitely don't want users editing the generated fields docs. And I'm pretty sure we don't want users editing the YAML files (where the source lives)--at least, not through the GitHub Edit link--so disabling the links is probably the best approach.